src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SNI/SNIHandle.cs

```diff
@@ -36,7 +36,7 @@ protected static async Task AuthenticateAsClientAsync(SslStream sslStream, strin
             {
                 TargetHost = serverNameIndication,
                 ApplicationProtocols = s_tdsProtocols,
-                EnabledSslProtocols = s_supportedProtocols,
+                EnabledSslProtocols = SslProtocols.None,
```

Comment:
None is the default so we can simply remove the line.
```suggestion
```

Reply:
done. I changed it in the original code since I was changing the comment.